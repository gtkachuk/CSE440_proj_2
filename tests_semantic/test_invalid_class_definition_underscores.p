program testInvalidClassDefinitionUnderscores;

class testInvalidClassDefinitionUnderscores

BEGIN

   VAR
      light_Switch	      : boolean;
      compiler_Works	      : boolean;

FUNCTION testInvalidClassDefinitionUnderscores: integer;
BEGIN
   lightSwitch := compilerWorks;
END

END
.
