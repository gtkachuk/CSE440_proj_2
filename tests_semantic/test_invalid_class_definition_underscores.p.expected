'_' (0137): illegal character at line 8
8: syntax error at 'Switch'
Errors detected. Exiting.
